docs/ai/ai-providers.md

```diff
@@ -0,0 +1,110 @@
+# AI Providers Usage Overview
+
+!!! warning "Warning"
+
+    These APIs are early in their development. Please feel free to [open a feature request and file issues](https://github.com/Eventual-Inc/Daft/issues/new/choose).
+
+
+Daft separates AI function protocols from Provider APIs, enabling a uniform interface for running models on data. Providers make it easy to route requests to different hosts and configure API keys.
+
+## Usage
+
+```python
+import os
+import daft
+
+# Set OpenAI Provider for Global Daft Session
+daft.set_provider(
+    "openai",
+    api_key=os.environ.get("OPENAI_API_KEY"),
+    timeout=30.0,
+    max_retries=3
+)
+
+# Retrieve Provider for an AI Function with
+daft.get_provider("openai")
+```
+
+Supported [AI Providers](../api/ai.md)
+
+- OpenAI
+- Transformers (+ Sentence Transformers)
+- LMStudio
```

Comment:
Outbound links to these might be nice. And maybe a one-sentence blurb would be useful for LMStudio